tools/release.sh

```diff
@@ -187,6 +187,9 @@ while ! openssl dgst -sha256 -verify $RELEASE_OPENSSL_PUBKEY -signature \
    read -p "Please correctly sign letsencrypt-auto with offline-signrequest.sh"
 done
 
+# This signature is not quite as strong, but easier for people to verify out of band
+gpg -u "$RELEASE_GPG_KEY" --detach-sign --armor --sign letsencrypt-auto-source/letsencrypt-auto
+
 # copy leauto to the root, overwriting the previous release version
 cp -p letsencrypt-auto-source/letsencrypt-auto certbot-auto
 cp -p letsencrypt-auto-source/letsencrypt-auto letsencrypt-auto
```

Comment:
As the tree and this PR stand, we'll be making the following signatures:

`letsencrypt-auto-source/letsencrypt-auto.sig` (openssl)
`letsencrypt-auto-source/letsencrypt-auto.asc` (gpg)

I guess we can't rename the former, because old `letsencrypt-auto`s will fetch it.  But we could probably rename the latter...